### DISCUSSION TITLE
using libreoffice plugins with code

### DISCUSSION THREAD

**Discussion by phirestalker** [CollaboraOnline/online#3663] [Q&A] [answered]
I would like to know if it is possible to use oxt plugins with CODE. I tried to install one through the command line, but it gave me a java error. I found that online is compiled without java. I am unsure how to continue.

I think I need to clone the online repository and edit the distribution configuration file for online to remove --without-java and then run the script to compile a nightly version of the docker images. Is there an easier way?

Thanks.

**Comment by pedropintosilva** [comment, marked as answer]
answered in the forum : https://forum.collaboraonline.com/t/using-extensions-in-collabora-online/734
